eclipse/dialog_settings.xml

```diff
@@ -3,7 +3,8 @@
 	<section name="SourceActionDialog.methods">
 		<item value="true" key="IgnoreDefault"/>
 		<item value="false" key="SynchronizedModifier"/>
-		<item value="false" key="Comments"/>
+		<item value="true" key="Comments"/>
+		<item value="1" key="InsertPosition.v2"/>
```

Comment:
I'd leave this unchanged as well; I don't usually like this after JavaBean accessor methods.